_source/reference/frames.md

```diff
@@ -87,6 +87,14 @@ Like an eager-loaded frame, but the content is not loaded from `src` until the f
 </turbo-frame>
 ```
 
+## Frame that will get reloaded with morphing during page refreshes
+
+```html
+<turbo-stream action="refresh">
```

Comment:
I'm fairly new to Turbo so apologies if I'm off base here, but this example is the only one to use `<turbo-stream>` rather than `<turbo-frame>`. It seems incomplete, perhaps.

Reply:
[Fixed here](https://github.com/hotwired/turbo-site/pull/161). Thanks 🙏